src/сomponents/AddTodo.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  filteredTodosCount: number;
+  todos: Todo[];
+  createTodo: (newTodo: string) => Promise<void>;
+  changeTodo: (todo: Todo) => void;
+  setError: (error: string) => void;
+};
+
+export const AddTodo: React.FC<Props> = ({
+  filteredTodosCount,
+  todos,
+  createTodo,
+  changeTodo,
+  setError,
+}) => {
+  const [title, setTitle] = useState('');
+  const [disableInput, setDisableInput] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    setDisableInput(true);
+    const newTitle = title.trim();
+
+    if (!newTitle) {
+      setError('Title can\'t be empty');
```

Comment:
Here is small hint. You can use the double quotes for such situations:
```
"Title can't be empty"
```
It looks much better as for me.